### PR TITLE
capssc: Add CaptureJpegScreenShot

### PR DESCRIPTION
**Pull request by SciresM**

### CODE REVIEW

File: nx/include/switch/services/capssc.h [hunk 1]
```diff
@@ -23,12 +24,22 @@ Service* capsscGetServiceSession(void);
...
  * @param height Image height, must be 720.
  * @param buffer_count Total number of output image buffers.
  * @param buffer_index Starting image buffer index. Must be < buffer_count.
  * @param timeout Timeout in nanoseconds. A default value of 100000000 can be used.
  */
-Result capsscCaptureRawImageWithTimeout(void* buf, size_t size, u32 layer_stack, u64 width, u64 height, s64 buffer_count, s64 buffer_index, u64 timeout);
+Result capsscCaptureRawImageWithTimeout(void* buf, size_t size, ViLayerStack layer_stack, u64 width, u64 height, s64 buffer_count, s64 buffer_index, s64 timeout);
 
+/**
+ * @brief This takes a screenshot, with the screenshot being written as jpeg into the ouput buffer.
```
Comment (fincs) [inline nx/include/switch/services/capssc.h:37]: ouput -> output

Comment (cathery) [reply nx/include/switch/services/capssc.h:37]: FIY you can use github suggestions to fix these errors yourself

File: nx/include/switch/services/capssc.h [hunk 2]
```diff
@@ -23,12 +24,22 @@ Service* capsscGetServiceSession(void);
  * @note buffer_index and buffer_count correspond to buffers with size 0x384000(1280*720*4). These must not be negative.
  * @param buf Output buffer containing the RGBA8 image.
  * @param size Size of buf, should be 0x384000(1280*720*4) * buffer_count.
- * @param layer_stack Value 0 can be used for this.
+ * @param layer_stack See \ref ViLayerStack.
```
Comment (yellows8) [inline nx/include/switch/services/capssc.h:27]: just `\ref ViLayerStack`

File: nx/include/switch/services/capssc.h [hunk 3]
```diff
@@ -26,7 +26,7 @@ Service* capsscGetServiceSession(void);
  * @note buffer_index and buffer_count correspond to buffers with size 0x384000(1280*720*4). These must not be negative.
  * @param buf Output buffer containing the RGBA8 image.
  * @param size Size of buf, should be 0x384000(1280*720*4) * buffer_count.
- * @param layer_stack See \ref ViLayerStack.
+ * @param layer_stack \ref ViLayerStack.
```
Comment (yellows8) [inline nx/include/switch/services/capssc.h:29]: remove period (same below)

**Pull request merged**